test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -189,6 +207,22 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			Entry("[test_id:45024] verify even split between 2 numa nodes", int32(1), int32(1)),
 		)
 
+		AfterAll(func() {
```

Comment:
[AfterAll] setup nodes must appear inside an Ordered container 
This means you will need to add Ordered to `Context("Verify hugepages count split on two NUMA nodes", func()`

Reply:
Fixed in the latest commit